docs/src/components/home/MaterialDesignComponents.tsx

```diff
@@ -36,85 +34,88 @@ import VerifiedUserRounded from '@mui/icons-material/VerifiedUserRounded';
 import HelpCenterRounded from '@mui/icons-material/HelpCenterRounded';
 import ROUTES from 'docs/src/route';
 import Link from 'docs/src/modules/components/Link';
+import { getDesignTokens, getThemedComponents } from 'docs/src/modules/brandingTheme';
 
-const Grid = styled('div')(({ theme }) => ({
-  borderRadius: theme.shape.borderRadius,
-  backgroundColor:
-    theme.palette.mode === 'dark'
-      ? theme.palette.background.paper
-      : alpha(theme.palette.grey[50], 0.4),
-  display: 'grid',
-  gridTemplateColumns: '1fr',
-  gridAutoRows: 240,
-  [theme.breakpoints.up('sm')]: {
-    gridAutoRows: 260,
-    paddingTop: 1,
-    gridTemplateColumns: '1fr 1fr',
-  },
-  [theme.breakpoints.up('md')]: {
-    gridAutoRows: 280,
-    gridTemplateColumns: '1fr 1fr 1fr',
-  },
-  '& > div': {
-    padding: theme.spacing(2),
-    alignSelf: 'stretch',
-    border: '1px solid',
-    borderColor:
-      theme.palette.mode === 'dark'
-        ? alpha(theme.palette.primaryDark[600], 0.3)
-        : theme.palette.grey[200],
-    [theme.breakpoints.only('xs')]: {
-      '&:first-of-type': {
-        borderTopLeftRadius: theme.shape.borderRadius,
-        borderTopRightRadius: theme.shape.borderRadius,
-      },
-      '&:last-of-type': {
-        borderBottomLeftRadius: theme.shape.borderRadius,
-        borderBottomRightRadius: theme.shape.borderRadius,
-      },
-      '&:not(:first-of-type)': {
-        marginTop: -1,
-      },
-    },
-    [theme.breakpoints.only('sm')]: {
-      marginTop: -1,
-      '&:first-of-type': {
-        borderTopLeftRadius: theme.shape.borderRadius,
-      },
-      '&:last-of-type': {
-        borderBottomRightRadius: theme.shape.borderRadius,
-        borderStyle: 'dashed',
-      },
-      '&:nth-of-type(even)': {
-        marginLeft: -1,
-      },
-      '&:nth-last-of-type(2)': {
-        borderBottomLeftRadius: theme.shape.borderRadius,
-      },
-      '&:nth-of-type(2)': {
-        borderTopRightRadius: theme.shape.borderRadius,
-      },
+const Grid = styled('div')(({ theme }) => [
+  {
+    borderRadius: (theme.vars || theme).shape.borderRadius,
+    backgroundColor: alpha(theme.palette.grey[50], 0.4),
```

Comment:
Should we get rid of the `alpha` usage?

Reply:
I don't think so because, for `grey` palette, there are no channel colors.